_std/defines/dialogue.dm

```diff
@@ -0,0 +1,2 @@
+#define DIALOGUE_CLOSE 1
```

Comment:
i'd move this to a `npc.dm` for generalization purposes

Reply:
i think this is specifically for the dialogue datum system? which is used for some consoles/object interactions / npcs? thats why i called it dialogue

Reply:
i think it's only used for npcs right now, just having 2 things in a file kinda hurts, could abstract into an `adventure.dm` maybe